content/docs/0.16.x/reference/cli/commands/keptn_install.md

```diff
@@ -11,7 +11,7 @@ Installs Keptn on a Kubernetes cluster
 
 The Keptn CLI allows installing Keptn on any Kubernetes derivative to which your kube config is pointing to, and on OpenShift.
 
-For more information, please follow the installation guide [Install Keptn](../../../../operate/install/#install-keptn)
+For more information, please follow the installation guide [Install Keptn](https://keptn.sh/docs/0.16.x/operate/install/#install-keptn)
```

Comment:
This should reference https://keptn.sh/docs/install/ .  It didn't get caught by CI because this is the correct location for 0.16.x but we have this new section in 0.17.x.